### PR TITLE
Feature/otro

### PR DESCRIPTION
**Pull request by LauraLozano**

### CODE REVIEW

File: algo.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def x():
+a=x
```
Comment (adminMesfix) [inline algo.py:2]: E112 expected an indented block

Comment (adminMesfix) [inline algo.py:2]: E225 missing whitespace around operator

Comment (adminMesfix) [inline algo.py:2]: E901 IndentationError: expected an indented block

Comment (adminMesfix) [inline algo.py:2]: W291 trailing whitespace

File: algo.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+def x():
+a=x 
+
+def y():
```
Comment (adminMesfix) [inline algo.py:4]: E302 expected 2 blank lines, found 1

File: algo.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+a=y
```
Comment (adminMesfix) [inline algo.py:5]: E112 expected an indented block

Comment (adminMesfix) [inline algo.py:5]: E225 missing whitespace around operator

**Comment by LauraLozano** [comment]
:+1: No lint errors found.

**Pull request closed without merging**